src/game.cpp

```diff
@@ -11349,6 +11355,7 @@ bool game::unload( item &it )
             u.moves -= mv;
             return true;
         } ), target->contents.end() );
+        target->on_contents_changed();
```

Comment:
Why this call? `magzines != containers` and `target` is guaranteed to be the former. It would be best not to confound these two systems (although in the long-run I agree they could be merged).